### PR TITLE
feat(ios): add allowsAirPlayForMediaPlayback prop

### PR DESCRIPTION
**Pull request by matinzd**
This PR adds `allowsAirPlayForMediaPlayback` props for iOS and MacOS platforms.  

Example:
```js
<WebView source={{ uri: 'https://reactnative.dev/' }} allowsAirPlayForMediaPlayback={false}  />
```

### CODE REVIEW

File: example/examples/NativeWebpage.tsx [hunk 1]
```diff
@@ -11,10 +11,13 @@ export default class NativeWebpage extends Component<Props, State> {
 
   render() {
     return (
-      <View style={{height: 400}}>
+      <View style={{ height: 400 }}>
         <WebView
-          source={{uri: 'https://infinite.red'}}
-          style={{width: '100%', height: '100%'}}
+          source={{
```
Comment (chiaramooney) [inline example/examples/NativeWebpage.tsx:16]: Are these changes to the example intentional? I think we'll want to keep the sample as it was.

Comment (matinzd) [reply example/examples/NativeWebpage.tsx:16]: It was due to my prettier. Maybe the author haven’t run prettier before on this file.

Comment (chiaramooney) [reply example/examples/NativeWebpage.tsx:16]: It looks like the url has been changed and your prop additional is still there. We might want to revert those!

Comment (matinzd) [reply example/examples/NativeWebpage.tsx:16]: Oops, I forgot to remove that one. Thanks for your note.

Comment (matinzd) [reply example/examples/NativeWebpage.tsx]: Reverted.

**Review by chiaramooney — APPROVED** [review]
👍

**Review by peterklingelhofer — COMMENTED** [review]
Just some minor suggestions regarding alphabetizing types. Both look like they could also be made into enums as well, potentially.

File: src/WebViewTypes.ts [hunk 2]
```diff
@@ -326,15 +331,19 @@ export interface AndroidNativeWebViewProps extends CommonNativeWebViewProps {
...
   forceDarkOn?: boolean;
   minimumFontSize?: number;
 }
 
-export declare type ContentInsetAdjustmentBehavior = 'automatic' | 'scrollableAxes' | 'never' | 'always';
+export declare type ContentInsetAdjustmentBehavior =
+  | 'automatic'
+  | 'scrollableAxes'
+  | 'never'
+  | 'always';
```
[the remark this answers is not in the document]
Comment (matinzd) [reply src/WebViewTypes.ts:346]: I didn’t make these changes and it’s not related to the PR.

File: src/WebViewTypes.ts [hunk 3]
```diff
@@ -12,25 +12,30 @@ import {
...
-type WebViewCommands = 'goForward' | 'goBack' | 'reload' | 'stopLoading' | 'postMessage' | 'injectJavaScript' | 'loadUrl' | 'requestFocus';
+type WebViewCommands =
+  | 'goForward'
+  | 'goBack'
+  | 'reload'
+  | 'stopLoading'
+  | 'postMessage'
+  | 'injectJavaScript'
+  | 'loadUrl'
+  | 'requestFocus';
```
[the remark this answers is not in the document]
Comment (matinzd) [reply src/WebViewTypes.ts:23]: I didn’t make these changes and it’s not related to the PR. These are just prettifier changes.

File: src/WebViewTypes.ts [hunk 4]
```diff
@@ -518,6 +529,11 @@ export interface IOSWebViewProps extends WebViewSharedProps {
    * @platform ios
    */
   allowsInlineMediaPlayback?: boolean;
+  /**
+   * A Boolean value indicating whether AirPlay is allowed. The default value is `false`.
+   * @platform macos
```
Comment (TheAlmightyBob) [inline src/WebViewTypes.ts:534]: Shouldn't that `@platform` be `ios`?

**Review by TheAlmightyBob — CHANGES_REQUESTED** [review]
I'm generally not a fan of including unrelated formatting changes... even if they're good changes, it would be clearer for reviews/history to have them in their own PR...

However, these are relatively minor and I see that my editor would make the same changes... so I guess not a huge deal

File: src/WebViewTypes.ts (hunk 4 above)
Comment (jamonholmgren) [reply src/WebViewTypes.ts:536]: @matinzd ^

Comment (matinzd) [reply src/WebViewTypes.ts]: Done.

**Review by jamonholmgren — APPROVED** [review]
Thanks @matinzd !

**Comment by jamonholmgren** [comment]
@TheAlmightyBob Can you remove your change request so we can merge & publish this?

**Pull request merged**